src/app/[locale]/layout.tsx

```diff
@@ -0,0 +1,70 @@
+import { dir } from 'i18next';
+import type { Metadata, Viewport } from 'next';
+import { Urbanist } from 'next/font/google';
+import { draftMode } from 'next/headers';
+
+import { ContentfulPreviewProvider } from '@src/components/features/contentful';
+import TranslationsProvider from '@src/components/shared/i18n/TranslationProvider';
+import { Footer } from '@src/components/templates/footer';
+import { Header } from '@src/components/templates/header';
+import initTranslations from '@src/i18n';
+import { locales } from '@src/i18n/config';
+
+export async function generateMetadata() {
+  const metatadata: Metadata = {
+    metadataBase: new URL(process.env.NEXT_PUBLIC_BASE_URL!),
+    other: {
+      'msapplication-TileColor': '#ffffff',
+      'msapplication-config': '/favicons/browserconfig.xml',
```

Comment:
This is IE, right? Can we delete that stuff?

Reply:
I think this is tied to adding a website to your Windows start menu, maybe?

Reply:
https://learn.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/samples/dn455106(v=vs.85)

Seems to be only for IE11, not for edge. Introduced for the windows 8 tiles. I think we can remove that without affecting anyone.